frontend/ui/data/dictionaries.lua

```diff
@@ -1195,6 +1195,2750 @@ local dictionaries = {
         license = "",
         url = "https://gitlab.com/avsej/dicts-stardict-form-xdxf/raw/d636cc5e8d4a47e22ac7466f4af6d435a8a3f650/002c/stardict-comn_sdict05_brokg-2.4.2.tar.gz"
     },
+    {
+        name = "Abkhaz-English Wiktionary dictionary",
```

Comment:
For the name I'd just keep it as Wiktionary for all of them:
```suggestion
        name = "Abkhaz-English Wiktionary",
```
After all, this is displayed in the dictionary download dialog.